### DISCUSSION TITLE
Suggestion about adtional notes for Wiki page: "FastAPI in Containers - Docker" -> "Prometheus and Other Reasons"

### DISCUSSION THREAD

**Discussion by boholder** [fastapi/fastapi#11241] [Questions] [answered]
https://fastapi.tiangolo.com/pl/deployment/docker/#prometheus-and-other-reasons

As the wiki says:

> In this case, if you had multiple containers, **by default,** when Prometheus came to read the metrics, it would get the ones for a single container each time (for the container that handled that particular request), instead of getting the accumulated metrics for all the replicated containers.

But Prometheus [supports](https://prometheus.io/docs/prometheus/latest/configuration/configuration) service discovery with various configuration sections that named as `***_sd_config`. By this way you can let Prometheus to fetch every node's metrics since Prometheus knows exact IPs of nodes.

I think the only **"by default,"** words isn't provides enough information for readers, which may makes them wrongly give up on cluster-level solution due to the description of wiki.

**Comment by YuriiMotov** [comment, marked as answer]
Seems that the docs has been updated and this is not relevant anymore..
